e2e-tests/playwright/tests/functional/channels/scheduled_drafts/create_scheduled_draft.spec.ts

```diff
@@ -0,0 +1,222 @@
+import {expect, Page} from '@playwright/test';
+import {test} from '@e2e-support/test_fixture';
+import {duration, wait} from '@e2e-support/util';
+import {ChannelsPage, ScheduledMessagePage} from '@e2e-support/ui/pages';
+
+test('Should create a scheduled message from a channel', async ({pw, pages}) => {
```

Comment:
Once available, please update this test and the next one with corresponding test IDs.

Reply:
Done. This PR - https://github.com/mattermost/mattermost/pull/28918 does vi via dev mode.